de.dlr.sc.virsat.model.extension.statemachines.ui/src/de/dlr/sc/virsat/model/extension/statemachines/ui/diagram/features/label/MultilineLabelFormatter.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.model.extension.statemachines.ui.diagram.features.label;
+
+import org.apache.commons.lang.WordUtils;
+
+/**
+ * Formating a multi line label
+ *
+ */
+public class MultilineLabelFormatter {

```

Comment:
Lets move the formatter to the model plugin, then we can also test it.
I dont see any UI dependencies.